backend/databaseUtils/adminPanel.js

```diff
@@ -0,0 +1,92 @@
+/* eslint-disable camelcase */
+require('dotenv').config();
+const postgres = require('postgres');
+
+const sql = postgres(process.env.E2ETEST == '1' ?
+    process.env.E2ETEST_POSTGRES_URL :
+    process.env.BACKEND_POSTGRES_URL);
+
```

Comment:
I am pretty sure we don't want to create multiple `sql` instances, but import `sql` from `database.js` instead. This could lead into situation where there are too many connections to the database, since these instances do not co-operate with each other.

Reply:
removed it in new commit